concepts/Vulnerability_disclosure_and_embargo_policy/00004-Vulnerability_disclosure_and_embargo_policy.md

```diff
@@ -0,0 +1,251 @@
+your report.
+
+After the initial reply to your report, the security team  will endeavor to 
+keep you informed of the progress being made towards a fix and full 
+announcement. As recommended by 
+[RFPolicy](https://dl.packetstormsecurity.net/papers/general/rfpolicy-2.0.txt),
+ these updates will be sent at least every five working days.
+
+If you have not received a reply to your initial contact within 72 hours, or
+have not heard from the security team for the past five days, there are a few
+steps you can take (in suggested order):
+- Contact the current security coordinator (Nathaniel McCallum, aka
+[npmacallum](https://github.com/npmccallum)) directly
+- Contact the back-up contact (Mike Bursell, aka 
+[MikeCamel](https://github.com/mikecamel)) directly  
+  In both these cases, you can reach out
+  - by email
+  - by chat
+- Come talk to us on the [chat](https://chat.enarx.io).
+
+As a reminder, **when escalating in these venues, please do not discuss your
+issue.** 
+
+Lastly, should you need to transfer files to us in a private manner, please 
+use [Magic Wormhole](https://magic-wormhole.readthedocs.io/en/latest/) or 
+make them available over https.
+
+
+#### Disclosure policy
+The Enarx project has a 5 step disclosure process.
+1. Contact is established, a private channel created, and the security report 
+is received and is assigned a primary handler. 
+  This person will coordinate the fix and release process.
+2. The problem is confirmed and a list of all affected versions is determined.
+  2.1 If an embargo is needed (see below), details of the embargo are decided.
+3. Code is audited to find any potential similar problems.
+4. Fixes are prepared for all releases which are still under maintenance. 
+  4.1 In case of embargo, these fixes are not committed to the public 
+  repository but rather held in a private fork pending the announcement.
+5. The changes are pushed to the public repository and new builds are deployed.
+  An announcement is sent to Enarx public channels (chat, website), once
+  these are available.
+
+This process can take some time, especially when coordination is required 
+with maintainers of other projects or, for instance, with hardware vendors. 
+Every effort will be made to handle the bug in as timely a manner as possible, 
+however it is important that we follow the release process above to ensure 
+that the disclosure is handled in a consistent manner.
+
+Enarx does not provide financial rewards for security disclosures, but the 
+team is open to working with reporters of issues on publishing and attribution 
+of vulnerabilities and fixes. We welcome reports of issues to the project, 
+and want to ensure that the community is aware of and celebrates security 
+improvements.
+
+#### Embargoes
+While the Enarx project aims to follow the highest standards of transparency 
+and openness (cf. [Enarx Principles](https://github.com/enarx/enarx.github.io/wiki/Design-principles)), 
```

Comment:
new link is: https://github.com/enarx/enarx/wiki/Design-principles

Reply:
Thanks, correcting this.